sdks/python/apache_beam/examples/ml_transform/vocab_tfidf_processing.py

```diff
@@ -130,7 +130,7 @@ def preprocess_data(
   data_pcoll = (
       data_pcoll | beam.ParDo(MapTFIDFScoreToVocab(artifact_location)))
 
-  _ = (data_pcoll | beam.io.WriteToText(output_dir))
+  #   _ = (data_pcoll | beam.io.WriteToText(output_dir))
```

Comment:
Can we get rid of this comment?

Reply:
Actually, is there a reason we're not writing anymore?

Reply:
This should be uncommented. I forgot to remove it/